### DISCUSSION TITLE
Need Help Optimizing Website Speed for Custom Packaging Business

### DISCUSSION THREAD

**Discussion by TroyYates** [joomla/joomla-cms#44517] [General]
Hi everyone,

I run a website called [Houston Packaging Solution](https://custompackaginghouston.com/), where we specialize in providing custom packaging solutions for businesses. While our site is working fine overall, I’ve noticed that the page load speed isn’t where I want it to be. It’s affecting the user experience, especially for potential customers browsing on mobile devices.

I’m reaching out to see if anyone here has experience with optimizing website speed, particularly for e-commerce and service-oriented sites like mine. Here are a few specific issues I’m facing:

**Page Load Time:** It’s currently averaging around 4-5 seconds and I know this can be improved for better user retention.
**Image Optimization:** The product images are large and I’m not sure if they’re fully optimized for speed.
**Caching & Code Optimization:** I’m not sure if I’m using the most effective caching strategies or if there’s unnecessary code slowing down the site.
Any advice on tools, techniques, or strategies to improve website speed (especially for e-commerce) would be highly appreciated! If you’ve dealt with similar issues in the packaging or small business niche, even better!

Looking forward to your insights and suggestions!

Thanks so much

**Comment by Hackwar** [comment]
Hello @TroyYates, this issue tracker is to manage bugs and features in the Joomla CMS and is NOT for general enduser support. Please take a look at forum.joomla.org for general support. Since this isn't a bug report, I'm closing this here.

**Reply by TroyYates** [reply]
Thank you for the clarification. I apologize for any misunderstanding. I’ll take the discussion over to the Joomla forum for support with optimizing website speed for my custom packaging business. I appreciate your guidance on where to get help. If there are any specific settings or configurations I should look into for Joomla to improve website speed, I’d love to hear your suggestions!

**Reply by brianteeman** [reply]
I am confused. The site you referenced in the first post is a wordpress web site not a joomla website
